theme/components/dropdown.scss

```diff
@@ -2,23 +2,14 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 .ck-dropdown {
-	display: inline-block;
-	position: relative;
-
 	// A triangle displayed to indicate this is actually a dropdown.
 	&::after {
-		content: '';
-		width: 0;
-		height: 0;
 		border-style: solid;
 		border-width: .4em .4em 0 .4em;
 		border-color: ck-color( 'text', 30 ) transparent;
-		pointer-events: none;
 
-		position: absolute;
 		top: 50%;
```

Comment:
It's pretty obvious that the dropdown "triangle" should be in the middle of the dropdown. Shouldn't it land in the Wireframe?